atef/bin/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .main import main
+from . import main
```

Comment:
This is a functionally different import than before, right? (`atef.bin.main` module vs the contained `main` function). Is this just to remove the ambiguity of statements like `from atef.bin import main`? Does this import need to be here at all?

Reply:
It is different - and part of a weird bug (or lack of understanding on my part).

Importing `main.main` as `main` effectively shadowed the submodule, and I wasn't able to access `atef.bin.main.something_else`. I switched it up to this instead.

I think it's not a necessary import. My only reason for keeping it would be that it's nice to have something that tab-completes when you import a submodule.